x-pack/plugins/ml/public/application/model_management/models_list.tsx

```diff
@@ -842,6 +843,7 @@ export const ModelsList: FC<Props> = ({
       ) : null}
       {isAddModelFlyoutVisible ? (
         <AddModelFlyout
+          data-test-subj={'mlAddTrainedModelFlyout'}
```

Comment:
This component does not support this property. You need to add it to `AddModelFlyoutProps` and pass it to the underlying `EuiFlyout`.